js/script.js

```diff
@@ -19,4 +19,14 @@ function startGame(event) {
   playerTwoName.innerText = playerTwoInput.value;
   playerThreeName.innerText = playerThreeInput.value;
   startScreen.classList.add('hidden');
-};
\ No newline at end of file
+  populate();
+};
+
+function populate() {
```

Comment:
Your `populate` function seems to be multi-tasking in ways it shouldn't - it's pulling in the `data` that you need for your categories as well as sending that modified data to display on the DOM. Remember that we want to encapsulate the information for our program (like our data) and keep it separate from the functionality that deals with DOM manipulation. 

I would redo this so that your data is being pulled into the class that will be handling your data. The start of the game (so your `Game` method of `startGame`) should trigger methods that get the data, manipulate it as necessary (you should be doing some modifications so that categories aren't displayed in camelCase) and then _passes_ that modified data to your function of `populate` that will populate all of that information to the DOM.  `populate` should only deal with actually displaying information to the DOM.